### ISSUE TITLE
Users who log in successfully are not redirected.

### ISSUE THREAD

**Issue by x106x** [pwm-project/pwm#656]
**Describe the bug**
The bug shows up primarily in Chrome and has for many years.  Replicating the bug seems extremely difficult however once it occurs it seems to happen in normal browsing or incognito.  Creating a new profile will get around the bug.

The Original Code fails to gather nextURL.  Output in developer mode is error = false and nextURL = "".  I'm not sure why this happens or how to reproduce this specific issue.

            var loadFunction = function(data) {
                if (data['error'] == true) {
                    PWM_MAIN.getObject('password').value = '';
                    PWM_MAIN.showErrorDialog(data,{
                        okAction:function(){
                            setTimeout(function(){
                                PWM_MAIN.getObject('password').focus();
                            },50);
                        }
                    });
                    return;
                }
                console.log('authentication success');
                var nextURL = data['data']['nextURL'];
                if (nextURL) {
                    PWM_MAIN.goto(nextURL, {noContext: true});
                }
            };

**Expected behavior**
The expected behavior is always to load /private however the variable returns empty.  Instead I've used window.location.replace after the logged authentication success and this resolves my issue completely.

**Desktop (please complete the following information):**
 - OS: Windows 7, 8, 9, 10 at least
 - Browser Chrome in the last 6+ years

**Additional context**
We have thousands of users and only a small amount use this tool so I can't say how often this happens.  We have run into this enough that it's annoying but have not been able to replicate this as it seems to happen or not.

**Comment by jrivard** [comment]
PWM Version?  Server Logs?  Browser logs/or steps to reproduce this?  If you don't give enough information to reproduce your issue we can't help much.

**Comment by x106x** [comment]
> PWM Version? Server Logs? Browser logs/or steps to reproduce this? If you don't give enough information to reproduce your issue we can't help much.

Using the current build or builds as far back as 4 years ago. The server logs are not helpful as it only seems to affect some clients. It was extremely difficult to reproduce but once we could we continued testing until it was fixed. Creating a new chrome profile seemed to be one mechanism that could change the outcome. At this point I believe it resolved the issue. Clearing cache or incognito didn't seem to affect the issue. The issue would appear in edge, chrome and Firefox. The browser logs would show the next url content with no data so I assume that's why it would do nothing.
